README.md

```diff
@@ -2,18 +2,99 @@
 
 [![Build Status](https://travis-ci.org/nokia-wroclaw/innovative-project-jackdaw.svg?branch=master)](https://travis-ci.org/nokia-wroclaw/innovative-project-jackdaw)
 
-## Run
 
-`docker-compose up`
+## Table of contents
+1. Project goals
+2. Technologies
+3. Functionality of modules
+4. Installation
+5. Contributing
+
+
+## 1. Project goals
+The goal is to create PoC of platform for streaming calculation using Kafka and Docker in microservices architecture. This platform will allow to perform Big Data Calculation in Streaming mode. As a final product there will be created a web application responsible for data visualization.
+
+As an example of dataset we used one that represents Flights in Brasil from www.kaggle.com.
```

Comment:
I'd use a reference:

[kaggle](www.kaggle.com)

Reply:
"As a final product there will be created a web application responsible for data visualization."
imho replace with
"For the final product a web application responsible for data visualization will be created."

Reply:
"As an example of dataset we used one that represents Flights in Brasil from www.kaggle.com."
imho replace with
"We used a dataset representing flights from and to Brasil as an example."

Reply:
Even
"We used a [dataset](https://www.kaggle.com/ramirobentes/flights-in-brazil) representing flights from and to Brasil as an example."